Assets/Scripts/Model/Types/Player.cs

```diff
@@ -67,7 +67,7 @@ public string FormattedName
             get
             {
                 string color = IsHome ? "#ff0000" : "#0000ff";
-                return $"<{color}>{TextPanelHandler.SanitizeText(Name)}</color>";
+                return $"<color={color}>{TextPanelHandler.SanitizeText(Name)}</color>";
```

Comment:
Not really necessary as the shorter format is explicitly supported by Unity. But if you want the more formal XML-like syntax, that's fine :)

Reply:
If anything, the colours should be const values in a more central place since they're used in other places too.

Reply:
I prefer the explicit format as the other seems like a syntactically wrong one which is tackled by the parser despite being wrong.

Colors moved to settings.